.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# These commits will be ignored by the GitHub blame view.
+# The git blame CLI can ignore them as well by doing:
+#   git blame --ignore-revs-file .git-blame-ignore-revs <filepath>
+# or via global config:
+#   git config --global blame.ignoreRevsFile .git-blame-ignore-revs
+#
+# Changed Layout/DotPosition from leading to trailing
+3fc49229761517cf3dc6979018a9a2d84f733ece
```

Comment:
I think the SHA you want to ignore is actually `138f7f746f635567b521dc4158f93be471939fae` which is the SHA of the merge commit?

```suggestion
138f7f746f635567b521dc4158f93be471939fae
```